loccallibrary/src/services/author.service.ts

```diff
@@ -7,3 +7,11 @@ export const numberOfAuthors = async () => {
   const authors = await authorRepository.count();
   return authors;
 };
+
+export const listAuthors = async () => {
+  const authors = await authorRepository.find({
+    order: { firstName: "ASC" },
+    relations: ["books"],
+  });
+  return authors;
```

Comment:
Fix tương tựu các file bên dưới e nha
```suggestion
  return await authorRepository.find({
    order: { firstName: "ASC" },
    relations: ["books"],
  });
```